packages/jaeger-ui/src/components/TraceDiff/TraceDiffGraph/drawNode.css

```diff
@@ -74,6 +74,16 @@ limitations under the License.
   padding: 0.3rem 0.5rem 0.3rem 0.75rem;
 }
 
+.DiffNode--popover .DiffNode--copyIcon,
+.DiffNode:not(:hover) .DiffNode--copyIcon {
+  display: none;
+}
```

Comment:
Is the use of `:not(...)` necessary? Would something like the following work?

```css
.DiffNode .DiffNode--copyIcon { display: none; }
.DiffNode:hover .DiffNode--copyIcon { display: unset; }
```

Reply:
is there a reason to avoid `:not`?
I started using to avoid defining multiple styles to achieve one goal (hiding something).
I only discovered when working on #310 when I was styling based on both hover state and focus state in: `/packages/jaeger-ui/src/components/TraceDiff/TraceDiffGraph/TraceDiffGraph.css`
That seemed easier than defining multiple styles so I started applying to more places.

Reply:
Word.